src/main/java/calculator/Calclator.java

```diff
@@ -0,0 +1,13 @@
+package calculator;
+
+public class Calclator {
+    Spliter spliter = new Spliter();
```

Comment:
여기서 선언한 Spliter가 새로 바뀌거나 한다면 위처럼 가변 참조 필드로 두어도 되겠지만, 그것이 아니기 때문에 private final 걸어주면 좋을것같아요

Reply:
추가로 Calculator(오타났네용!)와 Splitter는 다른 역할이잖아요? 여기서 Splitter 를 직접 생성해주고 있는데 이렇게 하는 경우 "결합도가 높다" 라고 해요!
Calculator가 Splitter 의존성을 가지고 있다고 볼 수 있는데, 이런 경우에 DI 패턴이라는걸 쓸 수 있습니다!
Splitter 의존을 Calculator 안에서 생성하지 않고 밖에서  넘겨주는 "의존성 역전" 이라는 개념인데
이 경우에도 민정이 리뷰처럼 private final 로 걸고 생성자를 통해 Splitter를 받으면 결합도를 낮출 수 있을 것 같아요 ㅎㅎ 
`Calculator calculator = new Calculator(new Splitter())` 

```suggestion
    private final Spliter spliter;
    
    public Calculator(Splitter splitter){
           this.splitter = splitter;
    }
```

Reply:
혹시 private final로 걸어야하는부분이 spliter 인가요?ㅠ